65black/StackOverflow-lite
# #160369939 Clean and Lint Code

#### What does this PR do?
This PR ensures that all the responses in our controllers and routes are descriptive and non-redundant. For example, a response of `{ statusCode: 200, message: 'successful' }` has been changed to `{ status: 'success', message: 'Operation Completed Successfully!'}`.

The PR also ensures that the every line of code is linted following Airbnb's standards and that there are no syntax errors.

#### Description of Task to be completed?
Ensures all the responses from the routes and controllers are descriptive and well-structured.
Also ensures that code is completely linted.

All js files and endpoints edited.

#### How should this be manually tested?
After cloning the repo and installing all dependencies with `npm install`

- Open postman and make any request to any endpoint
- Note that the responses are now well structured.

#### What are the relevant pivotal tracker stories?
#160369936
